docs/server/project_setup.rst

```diff
@@ -139,6 +139,17 @@ project.
 Once you are done click on the **Save** button below the form to create the
 project and save its Pootle FS configuration.
 
+If not all of your project's language codes match those available in Pootle,
+then you must add language mapping configurations for those languages. You can
+do that by clicking on the **Languages** link that is displayed below your
+project form:
+
+.. image:: ../_static/project_form_bottom_links.png
+
+
+and then provide a different language code in the **Filesystem language code**
+field for those languages whose codes need to be mapped.
+
```

Comment:
I think the above really want an example of what we mean or why this is important.  Something like:

This allows you to maintain consistent language codes on Pootle that can map to any codes that you have on various projects.  Examples might be `af` (Afrikaans) on Pootle but `af-ZA` in version control.  W can now map `af` to `af-ZA`.  Other common mappings might be `fr-FR` to `fr` (French) and `es_ES` to `es` (Spanish).  Or `en_GB` to `en-GB` where we are using different separators.

Reply:
This explanation is added by a later commit.